packages/client/src/runtime/core/init/getEngineInstance.ts

```diff
@@ -14,7 +14,7 @@ import { validateEngineInstanceConfig } from './validateEngineInstanceConfig'
  * @param engineConfig
  * @returns
  */
-export function getEngineInstance({ copyEngine = true }: GetPrismaClientConfig, engineConfig: EngineConfig): Engine {
+export function getEngineInstance(_: GetPrismaClientConfig, engineConfig: EngineConfig): Engine {
```

Comment:
let's maybe remove the first argument if it's not used anymore

it's not an implementation of a generic interface, no reason to keep a parameter around if it's not used in a concrete function